CMakeLists.txt

```diff
@@ -741,7 +745,11 @@ find_feature(Pulse ${PULSE_FEATURE_TYPE} ${PULSE_FEATURE_PURPOSE} ${PULSE_FEATUR
 find_feature(Cups ${CUPS_FEATURE_TYPE} ${CUPS_FEATURE_PURPOSE} ${CUPS_FEATURE_DESCRIPTION})
 find_feature(PCSC ${PCSC_FEATURE_TYPE} ${PCSC_FEATURE_PURPOSE} ${PCSC_FEATURE_DESCRIPTION})
 
-find_feature(CJSON ${CJSON_FEATURE_TYPE} ${CJSON_FEATURE_PURPOSE} ${CJSON_FEATURE_DESCRIPTION})
+option(WITH_AAD "Compile with support for Azure AD authentication" ON)
+if (WITH_AAD)
+	find_feature(CJSON REQUIRED ${CJSON_FEATURE_PURPOSE} ${CJSON_FEATURE_DESCRIPTION})
```

Comment:
better switch to [find_package](https://cmake.org/cmake/help/latest/command/find_package.html)